layouts/case-studies/list.html

```diff
@@ -21,6 +21,11 @@
 		</div>
 	</div>
 </section>
+<style>
+    #video button:hover {
```

Comment:
```suggestion
    #video button:hover {
          transition: 0.25s ease-out;
```
A small transition would look cleaner

Reply:
yeah @Gauravpadam it looks good will add those